IDE/IAR/settings/wolfBoot.wsdt

```diff
@@ -0,0 +1,533 @@
+<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
Consider not tracking this folder `settings`.

Reply:
updated, removed `settings` and other local files